driver/usb_hid_keys.h

```diff
@@ -25,4 +25,11 @@
 #define USB_HID_KEY_MEDIA_MUTE 0xEF
 #define USB_HID_KEY_MEDIA_SLEEP 0xF8
 
+#define USB_HID_USAGE_MEDIA_PLAYPAUSE 0xCD
+#define USB_HID_USAGE_MEDIA_PREVIOUSSONG 0xB6
+#define USB_HID_USAGE_MEDIA_NEXTSONG 0xB5
+#define USB_HID_USAGE_MEDIA_VOLUMEUP 0xE9
+#define USB_HID_USAGE_MEDIA_VOLUMEDOWN 0xEA
+#define USB_HID_USAGE_MEDIA_MUTE 0xE2
```

Comment:
Can you please add a reference where these numbers are from?

Reply:
Added reference to official USB HID usage document: https://usb.org/sites/default/files/hut1_22.pdf